avm/src/vm/run.rs

```diff
@@ -99,6 +100,7 @@ pub fn exec(fp: &str, delete_after_load: bool) {
     let mut vm = VM::new();
     let start = EventEmit { id: i64::from(BuiltInEvents::START), payload: None };
     vm.add(start);
+    telemetry::log().await;
```

Comment:
Doesn't this only log on application start, not all of the various commands?

Reply:
Yes :/

Reply:
https://github.com/alantech/alan/pull/318#issuecomment-727245916